python/lsst/meas/astrom/matchOptimisticB.py

```diff
@@ -207,7 +223,7 @@ def filterStars(self, refCat):
         return refCat
 
     @pipeBase.timeMethod
-    def matchObjectsToSources(self, refCat, sourceCat, wcs, refFluxField):
+    def matchObjectsToSources(self, refCat, sourceCat, wcs, refFluxField, exposure, minDist):
```

Comment:
Necessary to propagate back the new maximum distance between source and reference